.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 
 node_js:
   - '6'
-  - '7'
+  - '8.9'
```

Comment:
this should be '8', and we should also add '9'.

Reply:
Updated.

Reply:
Still the same here, did you push?